synapse/models/doc.py

```diff
@@ -6,35 +6,35 @@
 
             ('doc:authorable', {
```

Comment:
Do we want to add a property to capture the files development state? For example alpha, beta, released for software. For reports draft, in review, released, etc?

Reply:
Maybe... I'm not sure that there's a coherent set of "states" that we could enumerate that applies correctly to ALL authorables (do not overfit to reports here). It's additive (and not something we had in the old model either) so feel free to create a discuss ticket.